filebeat/tests/system/test_prospector.py

```diff
@@ -174,3 +174,39 @@ def test_exclude_files(self):
         # Check that output file has the same number of lines as the log file
         assert 1 == len(output)
         assert output[0]["message"] == "line in log file"
+
+    def test_rotating_ignore_older_low_write_rate(self):
+        self.render_config_template(
+            path=os.path.abspath(self.working_dir) + "/log/*",
+            ignoreOlder="1s",
+            scan_frequency="0.1s",
+        )
+
+        os.mkdir(self.working_dir + "/log/")
+        testfile = self.working_dir + "/log/test.log"
+
+        proc = self.start_filebeat(debug_selectors=['*'])
+        time.sleep(1)
```

Comment:
To guarantee filebeat started as expected, you can use `self.log_contains(...)` as you can find here: https://github.com/pghalliday/beats/blob/596/filebeat/tests/system/test_prospector.py#L39